docs/includes/pre-install-hardware.txt

```diff
@@ -0,0 +1,12 @@
+* Two computers with memory and hard drives to use as the SecureDrop servers.
```

Comment:
For checklist language, replace every spelled number with a numeral, i.e. "one"--->"1"